stdlib/public/runtime/CMakeLists.txt

```diff
@@ -21,17 +21,12 @@ endif()
 list(APPEND swift_runtime_compile_flags
      "-D__SWIFT_CURRENT_DYLIB=swiftCore")
 
-set(swift_runtime_objc_sources)
-set(swift_runtime_unicode_normalization_sources)
-if(SWIFT_HOST_VARIANT MATCHES "${SWIFT_DARWIN_VARIANTS}")
-  set(swift_runtime_objc_sources
-      ErrorObject.mm
-      SwiftObject.mm
-      SwiftValue.mm
-      Remangle.cpp
-      Reflection.mm)
-else()
-endif()
+set(swift_runtime_objc_sources
+    ErrorObject.mm
+    SwiftObject.mm
+    SwiftValue.mm
+    Remangle.cpp
+    Reflection.mm)
```

Comment:
Because its only included in the `add_swift_library` call with `TARGET_SDKS ALL_APPLE_PLATFORMS`, and not in the `add_swift_library` call for all non-Apple ones. When building on Linux, the files are still included in this list variable, but the variable is never used.